Modules/TPC/src/Tracks.cxx

```diff
@@ -49,6 +49,11 @@ void Tracks::initialize(o2::framework::InitContext& /*ctx*/)
     getObjectsManager()->startPublishing(&hist2);
     getObjectsManager()->addMetadata(hist2.GetName(), "custom", "42");
   }
+
+  for (auto& histR : mQCTracks.getHistogramRatios1D()) {
+    getObjectsManager()->startPublishing(&histR);
+    getObjectsManager()->addMetadata(histR.GetName(), "custom", "51");
```

Comment:
Isn't this a leftover from some example? I guess it is not needed. Perhaps this should also be removed everywhere else.
```suggestion
```

Reply:
Yes, I agree, this is not needed.